stub-idp/src/main/java/uk/gov/ida/stub/idp/domain/EidasScheme.java

```diff
@@ -6,50 +6,54 @@
  * This can't be used to read path params directly as we are using params with dashes
  */
 public enum EidasScheme {
-    stub_country("stub-country"),
-    stub_cef_reference("stub-cef-reference"),
-
-    austria("austria"),
-    belgium("belgium"),
-    bulgaria("bulgaria"),
-    croatia("croatia"),
-    cyprus("cyprus"),
-    czech_republic("czech-republic"),
-    denmark("denmark"),
-    estonia("estonia"),
-    finland("finland"),
-    france("france"),
-    germany("germany"),
-    greece("greece"),
-    hungary("hungary"),
-    ireland("ireland"),
-    italy("italy"),
-    latvia("latvia"),
-    lithuania("lithuania"),
-    luxembourg("luxembourg"),
-    malta("malta"),
-    netherlands("netherlands"),
-    poland("poland"),
-    portugal("portugal"),
-    romania("romania"),
-    slovakia("slovakia"),
-    slovenia("slovenia"),
-    spain("spain"),
-    sweden("sweden"),
-    united_kingdom("united-kingdom"),
-    iceland("iceland"),
-    liechtestein("liechtestein"),
-    norway("norway");
+    stub_country("stub-country", true),
+    stub_country_unsigned_assertions("stub-country-unsigned-assertions", false),
+    stub_cef_reference("stub-cef-reference", true),
+
+    austria("austria", true),
+    belgium("belgium", true),
+    bulgaria("bulgaria", true),
+    croatia("croatia", true),
+    cyprus("cyprus", true),
+    czech_republic("czech-republic", true),
+    denmark("denmark", true),
+    estonia("estonia", true),
+    finland("finland", true),
+    france("france", true),
+    germany("germany", true),
+    greece("greece", true),
+    hungary("hungary", true),
+    ireland("ireland", true),
+    italy("italy", true),
+    latvia("latvia", true),
+    lithuania("lithuania", true),
+    luxembourg("luxembourg", true),
+    malta("malta", true),
+    netherlands("netherlands", true),
+    poland("poland", true),
+    portugal("portugal", true),
+    romania("romania", true),
+    slovakia("slovakia", true),
+    slovenia("slovenia", true),
+    spain("spain", false), // unsigned assertions
+    sweden("sweden", true),
+    united_kingdom("united-kingdom", true),
+    iceland("iceland", true),
+    liechtestein("liechtestein", true),
+    norway("norway", true);
 
     private String eidasSchemeName;
+    private boolean shouldSignAssertions;
 
-    EidasScheme(String eidasSchemeName) {
+    EidasScheme(String eidasSchemeName, boolean shouldSignAssertions) {
```

Comment:
maybe it would of been worth keeping the default constructor of `EidasScheme(String eidasSchemeName)` aswell and just setting the value to default to `true`?